settings_test.py

```diff
@@ -48,6 +48,11 @@
     },
 }
 
+# Cheat and change SERVICES_DATABASE['NAME'] to the test database name. Our
+# update service doesn't use django database connections so we need to override
+# that setting so that it points to the correct database.
+SERVICES_DATABASE['NAME'] = 'test_%s' % SERVICES_DATABASE['NAME']
```

Comment:
I knew this was naive but yeah, this isn't going to work at all. The problems are:
- `services/utils.py` builds `mypool` directly at import time using `SERVICES_DATABASES`
- `SERVICES_DATABASES` doesn't take into account the test db

Django doesn't really provide an API to retrieve the test db name either... It's going to be in `settings.DATABASES[x]['NAME']`... once tests have been initialized. Need to be smarter about this.

Reply:
Found a proper fix through pytest fixtures, implemented in https://github.com/mozilla/addons-server/pull/14851/commits/f3ed1cfa0a68e934efbfa2c8c9234cd2d3f2f711